core/src/test/java/io/questdb/test/griffin/engine/functions/bool/AllNotEqStrFunctionFactoryTest.java

```diff
@@ -76,6 +76,23 @@ public void testMatch() throws Exception {
         );
     }
 
+    @Test
+    public void testMatchVarcharColumn() throws Exception {
+        assertQuery(
+                "a\n" +
+                        "aaa\n" +
+                        "aaa\n" +
+                        "bbb\n" +
+                        "ccc\n" +
+                        "ccc\n",
+                "select * from tab where a <> all('{abc,xyz}'::text[])",
```

Comment:
what is this magic?

Reply:
another overload for (varchar, string_array). however, I'm not sure if it's useful